rest/importtest/import_test.go

```diff
@@ -35,6 +35,45 @@ func assertExpiry(t testing.TB, expected uint32, actual uint32) {
 	assert.True(t, base.DiffUint32(expected, actual) < 10, fmt.Sprintf("Unexpected difference between expected: %v actual %v", expected, actual))
 }
 
+// TestImportFeed validates basic feed-based import
+func TestImportFeed(t *testing.T) {
+
+	base.SkipImportTestsIfNotEnabled(t)
+
+	rtConfig := rest.RestTesterConfig{
+		SyncFn: `function(doc, oldDoc) { channel(doc.channels) }`,
```

Comment:
I assume you are defining a syncfn because it will undergo js sync function and not skip? I'm not sure if that's needed for this test, I'd document why this is.

Autoimport should be enabled by default as far as I can tell by the `RestTester`so you could use the default config.

Reply:
It turns out autoimport is only enabled by default for EE, so still need the explicit setting here for the time being.